src/Sulu/Bundle/MediaBundle/Media/ImageConverter/Transformation/Manager/ManagerInterface.php

```diff
@@ -9,13 +9,13 @@
  * with this source code in the file LICENSE.
  */
 
-namespace Sulu\Bundle\MediaBundle\Media\ImageConverter\Command\Manager;
+namespace Sulu\Bundle\MediaBundle\Media\ImageConverter\Transformation\Manager;
```

Comment:
I don't really like that namespace... Since we are already renaming we could move that to the `Sulu\Bundle\MediaBundle\Media\ImageConverter` namespace.